src/iop/basecurve.c

```diff
@@ -424,6 +449,13 @@ int gauss_reduce_cl(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece,
 }
 
 
+static float exposure_increment(float stops, int e, float fusion, float bias)
+{
+  float offset = (stops * fusion) * (bias - 1.0) / 2.0;
+  return powf(2.0f, stops * e + offset);
```

Comment:
You understand correctly, yes.

To explain a bit further my motivation: my flow without my patch often looks like:

1) Oh, the overall contrast is too strong, let's use exposure fusion. Adjust exposure and enable fusion. More often than not, I get incorrect overall lightness at this point:
2) Adjust exposure in exposure module
3) Hmm, the exposure is right but the global contrast is not
4) Change the exposure shift in basecurve.
5) Global contrast seems OK, but now the exposure is wrong: goto 2)

After the patch, this is simplified in two ways: first, all steps can be done with the same module, but more importantly I find the bias setting which doesn't change the overall lightness (typically close to 0), and then play with the shift slider to see how much I want to compress global contrast. The important point is that the final setting is done with just one slider, so it's much easier to find the right point: change, see if it's better. With two sliders, I didn't have this immediate before/after comparison, I had to ply with history stack.

With bias=-1, you can also use the opposite flow of the one you suggest: expose for shadows, and then adjust the highlights using exposure fusion. Depending on the image and the expected effect, it can also make sense.